packages/lib-classifier/src/plugins/drawingTools/models/tools/Tool/Tool.spec.js

```diff
@@ -22,22 +22,22 @@ describe('Model > DrawingTools > Tool', function () {
       {
         type: 'multiple',
         answers: ['apples', 'oranges', 'pears'],
-        required: '',
+        required: false,
```

Comment:
This line was initially written as `required: false` in https://github.com/zooniverse/front-end-monorepo/pull/1376, but changed to `required: ''` in https://github.com/zooniverse/front-end-monorepo/pull/1768. Why?

Lots of other spec files still use `required: ''` such as Mark.spec, DrawingTask.spec, Task.spec, TextTask.spec, AnnotationStore.spec, and more. So which is correct?

Reply:
Back in 2020, the project builder had a bug that converted all values to strings, including booleans from checkboxes. That was fixed a while ago. Checkboxes should be saved as true/false now.

Reply:
This is the PR that fixed `handleInputChange` in the project builder, so checkboxes have been fixed since October 2020. 

https://github.com/zooniverse/Panoptes-Front-End/pull/5815

Reply:
Shouldn't all other files (Mark.spec, DrawingTask.spec, Task.spec, TextTask.spec, AnnotationStore.spec, etc) be updated to `required: false` in this PR too?

Reply:
THis PR is agnostic about whether Panoptes stored `task.required` as a string or a boolean, so mocks can use either and should still be represented internally as booleans in the classifier.

Reply:
I've updated the tests here, where the tests expected `task.required` to be an empty string. Otherwise, they shouldbe unchanged.

Reply:
>Shouldn't all other files (Mark.spec, DrawingTask.spec, Task.spec, TextTask.spec, AnnotationStore.spec, etc) be updated to required: false in this PR too?

That's a good question. I'd say not in this PR, since the tests are passing and **a refactor must not change public tested behaviour**, but maybe those mocks could be updated as maintenance for the classifier package? Those mocks mirror how the project builder behaved when they were written (five years ago, maybe?) Since then, bugs in the project builder have been fixed, and the new editor has been written. The mocks are now out of step with the tasks and steps that are created by current project builder code.

Depending on how old live projects are, the mocks might also need to reflect both the state of the project builder now, and the state of the project builder five years ago, in order to catch regressions that break backwards compatibility.

Reply:
I think this is also an arugment for breaking tasks out into their own NPM package, so that the project builder can import and use them along with the classifier. At the moment, task code is being edited in at least three places: the old PFE workflow editor, the new Pages Editor, and the task models here in the monorepo.